src/bridge.h

```diff
@@ -52,6 +39,8 @@ struct BridgeSpec {
 	StringID transport_name[2];         ///< description of the bridge, when built for road or rail
 	std::vector<std::vector<PalSpriteID>> sprite_table; ///< table of sprites for drawing the bridge
 	uint8_t flags;                         ///< bit 0 set: disable drawing of far pillars.
+	BridgeSpecCtrlFlags ctrl_flags{}; ///< control flags
+	std::array<std::array<BridgePillarFlags, AXIS_END>, NUM_BRIDGE_PIECES - 1> pillar_flags{}; ///< bridge pillar flags, 6 for each axis.
```

Comment:
Is there a reason this is different than elsewhere?

```suggestion
	static_assert(BRIDGE_PIECE_HEAD == 6);
	std::array<std::array<BridgePillarFlags, AXIS_END>, BRIDGE_PIECE_HEAD> pillar_flags{}; ///< bridge pillar flags, 6 for each axis.
```